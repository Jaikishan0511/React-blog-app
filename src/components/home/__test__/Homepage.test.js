import React from "react";
import renderer from "react-test-renderer";

import Homepage from "../index";
describe("homepage", () => {
  test("snapshot renders", () => {
    const component = renderer.create(<Homepage />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
