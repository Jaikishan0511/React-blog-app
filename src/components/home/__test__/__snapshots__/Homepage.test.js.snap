// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`homepage snapshot renders 1`] = `
<div>
  <h1>
    Welcome to the new Webpage!
  </h1>
</div>
`;
